lib/js/components/ProgressBar/ProgressBar.vue

```diff
@@ -0,0 +1,330 @@
+<template>
+	<div
+		:class="{
+			progressBar: true,
+			'-primary': color === PROGRESS_BAR_COLORS.PRIMARY,
+			'-warning': color === PROGRESS_BAR_COLORS.WARNING,
+			'-fail': color === PROGRESS_BAR_COLORS.FAIL,
+			'-success': color === PROGRESS_BAR_COLORS.SUCCESS,
+			'-neutral': color === PROGRESS_BAR_COLORS.NEUTRAL,
+			'-info': color === PROGRESS_BAR_COLORS.INFO,
+			'-schemeMedium': colorScheme === PROGRESS_BAR_COLOR_SCHEMES.MEDIUM,
+			'-schemeMediumNeutral': colorScheme === PROGRESS_BAR_COLOR_SCHEMES.MEDIUM_NEUTRAL,
+			'-compact': layout === PROGRESS_BAR_LAYOUTS.COMPACT,
+		}"
+	>
+		<div class="progressBar__label">
+			<div class="progressBar__labelText">{{ labelText }}</div>
+			<div v-if="labelDataExist" class="progressBar__labelDataWrapper">
+				<div v-if="labelData" class="progressBar__labelData">{{ labelData }}</div>
```

Comment:
because label is rendered as inline text, `span` looks more semantic for me

Reply:
what about other `divs` inside `__labelDataWrapper`?

Reply:
Also, I hope these are not the first symptoms of Divitis.